fars/booking/models.py

```diff
@@ -165,10 +169,34 @@ def get_overlapping_bookings(self):
             )
         return list(overlapping)
 
+    # Override save() method to be able to create Google Calendar events
+    def save(self, is_repeated_booking=False, *args, **kwargs):
```

Comment:
You may want to consider doing the gcal sync asynchronously and after save, so that the operation passes even if gcal isn't working. It might get quite frustrating for future users of this when suddenly the gcal credentials are revoked and anybody that knows anything about it are already long gone.

Reply:
Yes I thought of that, but then I realized I didn't know how to do that in Python. Do you recommend using `requests-futures`?

Reply:
Yeah, the external services notification (for GeneriKey) does this (I think the comment here is obsolete since we added the Futures): https://github.com/Teknologforeningen/fars/blob/master/fars/booking/models.py#L60-L65

Reply:
Thanks, I will look into it!

Reply:
All GCal handling is now happening in separate threads.